kernel/vars.mli

```diff
@@ -190,8 +190,8 @@ open UVars
 
 (** Level substitutions for polymorphism. *)
 
-val subst_univs_level_constr : sort_level_subst -> constr -> constr
-val subst_univs_level_context : sort_level_subst -> Constr.rel_context -> Constr.rel_context
+val subst_univs_level_constr : ?subst_rel:(Sorts.relevance -> Sorts.relevance) -> sort_level_subst -> constr -> constr
+val subst_univs_level_context : ?subst_rel:(Sorts.relevance -> Sorts.relevance) -> sort_level_subst -> Constr.rel_context -> Constr.rel_context
```

Comment:
This API doesn't look right to me

Reply:
Should this be part of `sort_level_subst` then ?

Reply:
It's unused in this PR so IDK what it's for.

Reply:
Let me remove it from this PR then, there are other changes in sorts that didn't make the cut anyways